docs/DeveloperGuide.md

```diff
@@ -124,7 +124,7 @@ The nontrivial methods in `ExpenseList` class contain some sort of exception han
 
 The following diagram is a UML class diagram for `Expense` and `ExpenseList`:
 
-![ExpenseAndExpenseList.png](/docs/diagrams/images/ExpenseAndExpenseList.png "UML Class Diagram for Expense and ExpenseList")
+![ExpenseAndExpenseList.png](diagrams%2Fimages%2FExpenseAndExpenseList.png "UML Class Diagram for Expense and ExpenseList")
```

Comment:
When handling links to images, we should standardise the IntelliJ format, drag and drop the image into the markdown editor.